apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactNewEdit.js

```diff
@@ -137,21 +136,29 @@ function ContactNewEdit({ id, data, close }) {
 		!phone && setPhoneError(null);
 	}, [phone]);
 
-	// Put the "userId" in state for the AutoComplete to have it available
+	// if contactManager username is set, then get its corresponding userId when the component is mounted
 	useEffect(() => {
 		if (!username) {
-			// No contact manager selected, skip population
 			return;
 		}
 
 		getUserData({ username }).then(({ user }) => {
 			setUserId(user._id);
-			// To avoid having to write a custom handler, lets store the current username on a different place
-			// This way, we can preserve the current username whenever CM changes
-			// All of this because CM is { username: string } instead of { _id: string }
-			setCurrentUsername(user.username);
 		});
-	}, [username, getUserData, setUserId]);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
🤔

Reply:
had to add this override since eslint was asking to add `username` as a dependency to this hook. However, we don't need that since we don't need to run this hook once when the component is loaded

Reply:
Thanks for sharing @aleksandernsilva :blue_heart: